src/appkit.cs

```diff
@@ -13055,6 +13316,43 @@ interface NSSegmentedControl {
 		[Mac (10, 12, 2)]
 		[NullAllowed, Export ("selectedSegmentBezelColor", ArgumentSemantic.Copy)]
 		NSColor SelectedSegmentBezelColor { get; set; }
+
+		[Mac (10,13)]
+		[Export ("setToolTip:forSegment:")]
+		void SetToolTip ([NullAllowed] string toolTip, nint segment);
+
+		[Mac (10,13)]
+		[Export ("toolTipForSegment:")]
+		[return: NullAllowed]
+		string ToolTipForSegment (nint segment);
```

Comment:
Genuine question but should we take the initiative to name it `Tooltip` instead of `ToolTip`?